rclc/package.xml

```diff
@@ -2,21 +2,27 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>micro_ros_demos_rclc</name>
-  <version>0.0.1</version>
-  <description>Example of a publisher using int32</description>
-  <maintainer email="julianbermudez@eprosima.com">Julian Bermudez</maintainer>
+  <version>0.0.3</version>
+  <description>Examples using RCL+RCLC API</description>
+  <maintainer email="borjaouterelo@eprosima.com">Borja Outerelo</maintainer>
```

Comment:
Should we set another maintainer?

Reply:
Sure, I'm adding myself and please tell me if you want to add someone else @JanStaschulat @ralph-lange